### ISSUE TITLE
CeskaTelevize does not work anymore

### ISSUE THREAD

**Issue by jfstod** [ytdl-org/youtube-dl#30406]
## Checklist

- [ x] I'm reporting a broken site support
- [ x] I've verified that I'm running youtube-dl version **2021.12.17**
- [ x] I've checked that all provided URLs are alive and playable in a browser
- [ x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.ceskatelevize.cz/porady/12833268908-staroveke-rise/219382567680006/', '--verbose']
[debug] Encodings: locale cp1250, fs mbcs, out cp852, pref cp1250
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: ffmpeg 3.0.8
[debug] Proxy map: {}
[CeskaTelevizePorady] 219382567680006: Downloading webpage
ERROR: Unable to extract iframe player url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\ceskatelevize.py", line 285, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract iframe player url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


```

## Description

After redesign and new version of iVysilani of CeskaTelevize, downloading of videos does not work with "ERROR: Unable to extract iframe player url". For example URl  https://www.ceskatelevize.cz/porady/12833268908-staroveke-rise/219382567680006/

Thanks for updating of  youtube-dl

**Comment by dirkf** [comment]
See also #30151. Seems to have been fixed in yt-dlp.

**Comment by jfstod** [comment]
Thank you for your response.
I am sorry, but yt-dlp neither does not work.
There is a new version of iVysilani since Dec 6th and downloading is broken. All issues are older here.

**Comment by dirkf** [comment]
[Fixed](https://github.com/yt-dlp/yt-dlp/commit/443b21dc4e499286df3852f88fbd48e7f61b1f6a#diff-77727401d445bb2863b8509fa248a57e8a7d4e8aa71766d7ef269376b52c6229) but not yet in the released version?
```
$ python3.9 -m yt_dlp -F -v 'https://www.ceskatelevize.cz/porady/12833268908-staroveke-rise/219382567680006/'
[debug] Command-line config: ['-F', '-v', 'https://www.ceskatelevize.cz/porady/12833268908-staroveke-rise/219382567680006/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.12.01 [91f071af6] (source)
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Git HEAD: 0fcba15d5
[debug] Python version 3.9.7 (CPython 32bit) - Linux-4.4.0-210-generic-i686-with-glibc2.23
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Optional libraries: keyring, sqlite
[debug] Proxy map: {}
[debug] [CeskaTelevize] Extracting URL: https://www.ceskatelevize.cz/porady/12833268908-staroveke-rise/219382567680006/
[CeskaTelevize] 219382567680006: Downloading webpage
[CeskaTelevize] 219382567680006: Downloading webpage
[CeskaTelevize] 219382567680006: Downloading webpage
[CeskaTelevize] 219382567680006: Downloading JSON metadata
[CeskaTelevize] 219382567680006: Downloading JSON metadata
[CeskaTelevize] 219382567680006: Downloading MPD manifest
[CeskaTelevize] 219382567680006: Downloading JSON metadata
[CeskaTelevize] 219382567680006: Downloading JSON metadata
[CeskaTelevize] 219382567680006: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[download] Downloading playlist: Aztékové - Starověké říše | Česká televize
[CeskaTelevize] playlist Aztékové - Starověké říše | Česká televize: Collected 1 videos; downloading 1 of them
[download] Downloading video 1 of 1
[info] Available formats for 61924494877789336:
ID                  EXT RESOLUTION │   FILESIZE   TBR PROTO  │ VCODEC        VBR ACODEC     ABR     ASR MORE INFO
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
dash-main-1002-1504 m4a            │ ~ 34.18MiB   96k dash   │ audio only        mp4a.40.2  96k 44100Hz [ces] DASH audio, m4a_dash
hls-main-628        mp4 unknown    │ ~223.59MiB  628k m3u8_n │ unknown      628k unknown     0k
hls-main-1160       mp4 unknown    │ ~413.01MiB 1160k m3u8_n │ unknown     1160k unknown     0k
hls-main-2176       mp4 unknown    │ ~774.75MiB 2176k m3u8_n │ unknown     2176k unknown     0k
dash-main-1001-1502 mp4 512x288    │ ~189.41MiB  532k dash   │ avc1.4d4015  532k video only             DASH video, mp4_dash
dash-main-1001-1503 mp4 720x404    │ ~378.83MiB 1064k dash   │ avc1.4d401e 1064k video only             DASH video, mp4_dash
dash-main-1001-1504 mp4 1024x576   │ ~740.57MiB 2080k dash   │ avc1.4d401f 2080k video only             DASH video, mp4_dash
[download] Finished downloading playlist: Aztékové - Starověké říše | Česká televize
$
```
However extractor test 0 fails (`RegexNotFoundError: [CeskaTelevize] 20641-bonus-01-en: Unable to extract type`) and the metadata extraction fails in test 2 (`Invalid value for field title, expected 'Queer: Bogotart', got 'Bogotart - Queer | Česká televize'`).

**Comment by jfstod** [comment]
Thanks, I will wait for new version.
